oauth2/oauth2-resource-server/src/test/java/org/springframework/security/oauth2/server/resource/web/DefaultBearerTokenResolverTests.java

```diff
@@ -117,7 +118,18 @@ public void resolveWhenValidHeaderIsPresentTogetherWithFormParameterThenAuthenti
 	}
 
 	@Test
-	public void resolveWhenValidHeaderIsPresentTogetherWithQueryParameterThenAuthenticationExceptionIsThrown() {
```

Comment:
These tests seem to already line up well with my suggestion. The only thought I have is: Can we can keep the existing test names, and only have new names for truly new tests?